src/devices/bus/megadrive/rom.cpp

```diff
@@ -1667,3 +1668,939 @@ void md_rom_sram_arg96_device::write(offs_t offset, uint16_t data, uint16_t mem_
 		logerror("unhandled write at offset %08x %04x %04x\n", offset, data, mem_mask);
 	}
 }
+
+/*-------------------------------------------------
+ Paprium
+ Cortex-M4 CPU code isn't dumped, we have to make
+ up for the missing features with best guesses of
+ internal workings.
+
+ Current issues:
+ - First boot will launch a fake minigame, this is normal
+ when save is empty.
```

Comment:
Not an issue, it's part of the game (and afaik some MD HW revisions will get stuck over this minigame too).